extensions/auth/portability-auth-smugmug/src/main/java/org/dataportabilityproject/auth/smugmug/SmugMugAuth/SmugMugOauthInterface.java

```diff
@@ -55,7 +55,8 @@ public String getAuthorizationUrl(TokenSecretAuthData token, String permission)
     OAuthService service =
         new ServiceBuilder().provider(SmugMugOauthApi.class).apiKey(apiKey).apiSecret(apiSecret).build();
     String authUrl = service.getAuthorizationUrl(new Token(token.getToken(), token.getSecret()));
-    return String.format("%s&perms=%s", authUrl, permission);
+    // Request for permissions and for Full access (private and public content)
+    return String.format("%s&Permissions=%s&Access=%s", authUrl, permission, "Full");
```

Comment:
Will authUrl alrready contain a starting parameter, eg. ?something=something that can be appended to or do we need to check here?

Reply:
authUrl will already contain the oauth query params for the auth request and callback parameter